src/main/java/de/tum/in/www1/artemis/security/jwt/JWTCookieService.java

```diff
@@ -56,12 +58,13 @@ public ResponseCookie buildLogoutCookie() {
      */
     private ResponseCookie buildJWTCookie(String jwt, Duration duration) {
 
+        // TODO - Remove cypress workaround once cypress uses https and find a better solution for testing locally in Safari
         Collection<String> activeProfiles = Arrays.asList(environment.getActiveProfiles());
-        boolean isCypress = activeProfiles.contains(CYPRESS_PROFILE);
+        boolean isSecure = !activeProfiles.contains(CYPRESS_PROFILE) && !activeProfiles.contains(DEVELOPMENT_PROFILE);
 
         return ResponseCookie.from(JWT_COOKIE_NAME, jwt).httpOnly(true) // Must be httpOnly
                 .sameSite("Lax") // Must be Lax to allow navigation links to Artemis to work
-                .secure(!isCypress) // Must be secure - TODO - Remove cypress workaround once cypress uses https
```

Comment:
Should we keep this TODO for Cypress, e.g. in line 63 now?

Reply:
In our developer meeting on Friday, we agreed to add a check for the "dev" profile. This is only a temporary solution since we should not add checks for profiles in production code (we haven't found a better solution yet).

I moved the TODO to line 61, above the two lines responsible for the profile check